<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepositCalc</class>
 <widget class="QMainWindow" name="DepositCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Label_operMonth">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>in month</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="box_t_rate">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>211</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <double>0.010000000000000</double>
    </property>
    <property name="maximum">
     <double>10000.000000000000000</double>
    </property>
    <property name="value">
     <double>2.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="button_add_wthwal">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>270</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Add withdrawal</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_depoAmount">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>80</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Deposit amount by the end of the term</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_amount">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Deposit amount</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_operAmount">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Amount</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Label_perid">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Periodicity of payments</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="box_capitalzation">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>160</y>
      <width>84</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="display_total_int">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>20</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Label_rate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Interest rate (%)</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="box_oper_month">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>230</y>
      <width>211</width>
      <height>24</height>
     </rect>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100000.000000000000000</double>
    </property>
    <property name="value">
     <double>12.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="Label_capitalization">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Capitalization of interest</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_Calculate_Deposit">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>140</y>
      <width>481</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_tax">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tax rate  (%)</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="display_deposit_ops">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>220</y>
      <width>501</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="display_end_amount">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>80</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="box_dep_period">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>211</width>
      <height>24</height>
     </rect>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100000.000000000000000</double>
    </property>
    <property name="value">
     <double>12.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="box_i_rate">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>211</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <double>0.010000000000000</double>
    </property>
    <property name="maximum">
     <double>10000.000000000000000</double>
    </property>
    <property name="value">
     <double>5.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="Label_period_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Period (month)</string>
    </property>
   </widget>
   <widget class="QLabel" name="display_total_tax">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>50</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="CP_Periodicity">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>monthly</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>quaterly</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>annualy</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="Label_interestAccrued">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Accrued interest</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_plannedOper">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>190</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Planned operations</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="box_dep_amount">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>211</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <double>0.010000000000000</double>
    </property>
    <property name="maximum">
     <double>1000000000000000.000000000000000</double>
    </property>
    <property name="value">
     <double>1000.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="button_revert_op">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Revert last operation</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="box_oper_amount">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>211</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <double>0.010000000000000</double>
    </property>
    <property name="maximum">
     <double>1000000000000000.000000000000000</double>
    </property>
    <property name="value">
     <double>1000.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="button_add_repl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Add replenishment</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_taxAmount">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>50</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tax amount</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
